Makefile

```diff
@@ -129,6 +129,26 @@ test-coverage:
 	@export VERSION=$(VERSION); bash -x scripts/test_cover.sh
 .PHONY: test-coverage
 
+## txsim-install: Install the tx simulator/fuzzer.
+txsim-install:
+	@echo "--> Installing tx simulator"
+	@go build -mod=readonly ./test/cmd/txsim
+.PHONY: txsim-install
+
+## txsim-build: Build the txsim binary into the ./build directory.
+txsim-build: mod
+	@echo "--> Building tx simulator"
+	@cd ./test/cmd/txsim
+	@mkdir -p build/
+	@go build $(BUILD_FLAGS) -o build/ ./test/cmd/txsim
+	@go mod tidy -compat=1.20
+.PHONY: txsim-build
+
+## docker-txsim-build: Build the Docker container txsim .
```

Comment:
[nit] I think this makes a Docker image not a container

```suggestion
## docker-txsim-build: Build the Docker image txsim.
```

based on help docs

```
$ docker build --help

Usage:  docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile
```

Reply:
Ack